src/applications/claims-status/components/ClaimLetterListItem.jsx

```diff
@@ -14,8 +14,9 @@ const formatDate = buildDateFormatter(DATE_FORMATS.LONG_DATE);
 
 const docTypeToDescription = {
   27: 'Board Of Appeals Decision Letter',
-  65: '5103 Notice',
-  68: '5103 Notice',
+  704: '5103 Notice',
```

Comment:
Can you please add context about why we are removing 65 and 68 and adding 704, 706, and 858?

Reply:
@pmclaren19 There was context in the summary, but I just added specific references to the doctype numbers

Reply:
@pmclaren19 we only have approval from VBA to expose the docs that correspond specifically to the numerical codes 704, 706, and 858 (this one was not shown in that snapshot in the ticket, but is a type of 5103 notice).

Reply:
Talked to @jacobworrell in parking lot 

- letter id 704 correlates to 65 and we were just referencing the wrong id#. It is for **Standard 5103 Notice**
- letter id 706 correlates to 68 and we were just referencing the wrong id#. It is for **5103/ DTA letter**
- letter id 858 needed to be added and is for **Custom 5103 Notice**